wallet/wallet.h

```diff
@@ -34,6 +34,7 @@ struct wallet {
 	u64 max_channel_dbid;
 
 	struct outpointfilter *owned_outpoints;
+	struct outpointfilter *utxoset_outpoints;
```

Comment:
The relationship between these two is not immediately obvious to me?

owned_outputs is all outputs we own, including spent ones?  utxoset_outpoints is a complete UTXO set?  Or UTXOs corresponding to channel opens?

Reply:
Added a comment to each about what they include.